modules/articles.modules.js

```diff
@@ -9,4 +9,24 @@ exports.selectArticleById = (id) => {
         }
         return rows[0]
     })
+}
+
+exports.selectArticles = () => {
+    const queryString = `SELECT articles.author, 
+                                title,
+                                articles.article_id, 
+                                topic, 
+                                articles.created_at,
+                                articles.votes,
+                                article_img_url,
+                                COUNT(comments.article_id) AS comment_count
+                                FROM articles
+                                LEFT JOIN comments 
+                                ON articles.article_id = comments.article_id 
+                                GROUP BY articles.article_id
+                                ORDER BY articles.created_at DESC`
+    return db.query(queryString)
+    .then(({rows}) => {
+        return rows
```

Comment:
Again, we should have two models for the two end points. When returning just a singular article, we want to send back an object, for multiple articles we want to send back an array